conda-store-server/conda_store_server/app.py

```diff
@@ -201,7 +205,7 @@ class CondaStore(LoggingConfigurable):
     )
 
     database_url = Unicode(
-        "sqlite:///conda-store.sqlite",
+        "sqlite:///" + str(PurePosixPath(CONDA_STORE_DIR) / "conda-store.sqlite"),
```

Comment:
This needs to be verified and used everywhere.

```py
>>> str(PurePosixPath(CONDA_STORE_DIR) / "foo")
'C:\\/Users/nikita/.conda-store/foo'

>>> str(CONDA_STORE_DIR / "foo")
'C:\\Users\\nikita\\.conda-store\\foo'

>>> str(PosixPath(CONDA_STORE_DIR) / "foo")
NotImplementedError: cannot instantiate 'PosixPath' on your system
```

Here people say that it works fine with Windows-style paths:
https://stackoverflow.com/questions/19260067/sqlalchemy-engine-absolute-path-url-in-windows